src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
The `Cloneable` interface is implemented here, but there is no clear indication of why cloning is necessary for the `Country` entity. Entities in Hibernate are typically not cloned. If cloning is not a requirement, consider removing the `Cloneable` interface and the `clone` method to adhere to the principle of minimalism in design.